bentoml/_internal/external_typing/__init__.py

```diff
@@ -30,6 +32,8 @@
     from .starlette import ASGIReceive
     from .starlette import AsgiMiddleware
 
+    WSGIApp = t.Callable[[F, t.Mapping[str, t.Any]], t.Iterable[bytes]]
```

Comment:
Move WSGIApp from `bentoml._internal.service.service` to here to use in `bentoml_cli`